sql/base/auth_database.sql

```diff
@@ -2399,7 +2399,8 @@ INSERT INTO `updates` VALUES
 ('2020_12_31_00_auth.sql','05C9C105D55C6588CDA0D75AE3B135B7E6B54C06','RELEASED','2020-12-31 12:58:21',0),
 ('2021_01_10_00_auth.sql','4430F5CC9A31DBBEA4E437E980E9F26AC919C016','RELEASED','2021-01-10 12:29:57',0),
 ('2021_01_13_00_auth.sql','F7C15519FFA0FE27EED859343B58714624A302E6','RELEASED','2021-01-13 11:19:52',0),
-('2021_02_09_00_auth.sql','F00ABFF6E3C3F5ACE3444C9D70BADC764C8B3CE2','RELEASED','2021-02-09 17:22:24',0);
+('2021_02_09_00_auth.sql','F00ABFF6E3C3F5ACE3444C9D70BADC764C8B3CE2','RELEASED','2021-02-09 17:22:24',0),
+('2021_03_14_00_auth_2017_12_17_00_auth.sql','2CD99730D4D32DBF0584CD5B1AA6F8F4AE3DA975','RELEASED','2021-03-14 00:00:00',0);
```

Comment:
Auth files must not be renamed when cherry picking, keep original name

Reply:
how to ensure they are applied at last? i add prefix like in world

Reply:
order is less important than not applying the update twice (auth db is supposed to be shareable)

Reply:
Ok